backend/handlers/base_handler.py

```diff
@@ -14,7 +14,7 @@ class BaseHandler(webapp2.RequestHandler):
     def decorateHeaders(self):
         """Decorates headers for the current request."""
         self.response.headers['Access-Control-Allow-Origin'] = '*'
-        self.response.headers['Access-Control-Allow-Headers'] = 'X-API-KEY, Origin, X-Requested-With, Content-Type, Accept, Access-Control-Request-Method, Authorization'
+        self.response.headers['Access-Control-Allow-Headers'] = 'X-API-KEY, Origin, X-Requested-With, Content-Type, Accept, Access-Control-Request-Method, Authorization, Institution-Authorization'
```

Comment:
Curtir um post sem erro, mas comentar dá esse erro:
File "/home/mayza/workspace/web/backend/handlers/post_comment_handler.py", line 47, in post
    comment_data = body['commentData']
KeyError: 'commentData'

Reply:
:+1: